src/devicelock/devicelock.cpp

```diff
@@ -199,7 +199,9 @@ bool DeviceLock::runPlugin(const QStringList &args)
     qWarning() << process.readAllStandardError();
 #endif
 
-    return process.exitCode() == 0;
+    if (process.exitCode() == 0) return OK;
```

Comment:
If you change the enum, so that OK is 0 and Failed is 1, you don't need do have that thing here, and can just say `return process.exitCode();`